.github/semantic.yaml

```diff
@@ -0,0 +1,59 @@
+###############################################################################
+# This file configures "Semantic Pull Requests", which is documented here:
+# https://github.com/zeke/semantic-pull-requests
+###############################################################################
+
+# Scopes are optionally supplied after a 'type'. For example, in
+#
+# feat(cdr): autostart ui
+#
+# '(cdr)' is the scope. Scopes are used to signify where the change occurred.
```

Comment:
comment needs updating or removing depending on the outcome in https://github.com/coder/code-server/pull/4652/files#r771879992